.pre-commit-config.yaml

```diff
@@ -127,3 +127,13 @@ repos:
         -   "--ignore=tests/test_scripts.py"
         always_run: true
         pass_filenames: false
+
+-   repo: local
+    hooks:
+    -   id: deptry
+        name: deptry
+        stages: [commit, push, manual]
```

Comment:
```suggestion
        stages: [push, manual]
```

Reply:
or do you find this necessary pre-commit?

Reply:
> or do you find this necessary pre-commit?

No, not really. I think it is reasonable to run the hook on pushs only.

By the way, as of pre-commit [`3.2.0`](https://pre-commit.com/#confining-hooks-to-run-at-certain-stages) the stages `[commit, merge-commit, push, ...]` have been deprectaed (aliased to) `[pre-commit, pre-merge-commit, pre-push, ...]` respectively. We might want to start using these.